neurom/view/view.py

```diff
@@ -163,14 +158,38 @@ def plot_soma(ax, soma, plane='xy',
                                    ignore=False)
 
 
+def _get_scaled_linewidth(ax, scale_with=None):
+    """Get linewidth to scale diameters according to the axis of current figure.
+
+    WARNING: this works best if both axis are on the same scale, and will not follow
```

Comment:
ah yes, I should remove the 'axis on the same scale', this was for the first implementation I made, now it works independently of that. For your second point, fixing it would involve more coding (see the link in the main description of the PR), as lines need to be redrawn if the user rescale things on the fly... this option is not on by default, and should be used safely once we just plt.savefig. If the user start modifying the figure, diameter will not follow properly (as it has always been the case).